tests/scripts/bootstrap.sh

```diff
@@ -73,7 +73,7 @@ configure_network() {
     echo 1 | sudo tee /proc/sys/net/ipv4/conf/all/forwarding
 }
 
-case $TRAVIS_OS_NAME in
+case $OS_NAME in
```

Comment:
use a well-known environment?
https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script